### DISCUSSION TITLE
How to display user image?

### DISCUSSION THREAD

**Discussion by ayahaceh** [thedevdojo/voyager#5251] [Q&A]
Apps doesn show image as well. then I was re-upload image for user, It upload success, but it doent show in view.
How to fix this?
(Laravel 8)

![Voyage](https://user-images.githubusercontent.com/14037293/110238877-876f0300-7f76-11eb-931a-d359312decb8.png)

**Comment by mayronH** [comment]
Check in your .env if the APP_URL is correct, also try to run `php artisan storage:link`
